cloudbuild.yaml

```diff
@@ -118,6 +118,14 @@ steps:
   args: [ '-U', 'admin', '-d', 'postgres', '-h', 'hapi-fhir-db', '-p', '5432',
           '-c', 'CREATE DATABASE views;']
 
+- name: 'docker/compose'
```

Comment:
nit: I suppose you are doing this just to clear the content of the current FHIR sink server (which is fine). If that's the case, consider adding a short comment above, e.g., `# Resetting FHIR sink server` or something like that.

Reply:
done